Service/Adapter/Flysystem.php

```diff
@@ -9,19 +9,25 @@
 
 use Kitpages\FileSystemBundle\Model\AdapterFileInterface;
 use Kitpages\UtilBundle\Service\Util;
+use Techyah\Flysystem\OVH\OVHAdapter;
+use Terradona\Infrastructure\FileBundle\Adapter\OvhCustomAdapter;
 
 class Flysystem implements CmsAdapterInterface
 {
 
     private $filesystem;
+    private $util;
+    private $fileUriPrefix;
 
     public function __construct(
         Util $util,
-        Filesystem $filesystem
+        Filesystem $filesystem,
+        $fileUriPrefix = null
     )
     {
         $this->util = $util;
         $this->filesystem = $filesystem;
+        $this->fileUriPrefix = $fileUriPrefix;
```

Comment:
Ensure there is no trailing slash here as you append a forced one there https://github.com/kitpages/KitpagesFileSystemBundle/pull/5/files#diff-956f4bd530ea68b5e322e15b5626662cR121